iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -180,6 +244,35 @@ TEST_F(RoudiFindService_test, StopOfferMultiMethodServiceSingleInstance)
     receiverRuntime->findService({"service3", "instance1"}, instanceContainer);
     ASSERT_THAT(instanceContainer.size(), Eq(0u));
 }
+
+TEST_F(RoudiFindService_test, StopOfferServiceSingleMultipleCalls)
```

Comment:
Please let the name reflect what you test and what you expect.

Reply:
name changed